src/components/MovieDetailsContainer.js

```diff
@@ -1,11 +1,11 @@
-import React from 'react'
-import '../css/MovieDetailsContainer.css'
-import MovieDetailsCard from './MovieDetailsCard'
+import React from 'react';
+import '../css/MovieDetailsContainer.css';
+import MovieDetailsCard from './MovieDetailsCard';
 
 
 const MoviesDetailsContainer = ({ movieSelected, redirectHome, status }) => {
   if (status) {
```

Comment:
I've seen `status` referenced a couple times in the code, and at first glance I'm not sure what it's used for. Consider renaming the variable to something a little more explicit for what the status is.